overrides.go

```diff
@@ -0,0 +1,72 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+//
+// Contributor:
+// - Brandon Myers bmyers@mozilla.com
+
+package main
+
+import (
+  "fmt"
+  "os"
+  "bufio"
+  "strings"
+  "strconv"
+)
+
+type Override struct {
+  CIDR string
+  City string
+  Country string
+  Latitude float64
+  Longitude float64
+}
+
+
+func readOverrides(path string) (overrides []Override){
+  file, err := os.Open(path)
+  defer file.Close()
```

Comment:
This defer should be after checking the status of err (e.g., below `if err != nil`) otherwise we could call Close on an undefined value for file

Reply:
Ahh, good catch, I've implemented it in https://github.com/ameihm0912/geomodel/pull/10/commits/7924535a214c3750e5ab7e2b4d0e4058e63dc2c1